src/confluent_kafka/src/Consumer.c

```diff
@@ -980,6 +980,28 @@ static PyObject *Consumer_poll (Handle *self, PyObject *args,
 }
 
 
+static PyObject *Consumer_memberid (Handle *self, PyObject *args,
+                                    PyObject *kwargs) {
+        char *memberid;
+        PyObject *memberidobj;
+        if (!self->rk) {
+                PyErr_SetString(PyExc_RuntimeError,
+                                "Consumer closed");
+                return NULL;
+        }
+
+        memberid = rd_kafka_memberid(self->rk);
+
+        if (!memberid)
+                Py_RETURN_NONE;
+
+        memberidobj = Py_BuildValue("s", memberid);
```

Comment:
librdkafka will return an empty string rather than null if it hasn't received a memberid from the coordinator yet, but I think it makes more sense to return None from the python bindings in that case, so recommend adding that.
E.g.,;

Reply:
Nevermind, it looks like it'll actually return NULL

Reply:
It is returning empty string.